### ISSUE TITLE
Missing libmonitoring-plugin-perl for crm/corosync checks on jammy

### ISSUE THREAD

**Issue by sudeephb** [canonical/charm-nrpe#76]
When trying to run /usr/local/lib/nagios/plugins/check_crm or check_corosync_rings on jammy machines, I got the following error:
Can't locate Nagios/Plugin.pm in **INC** (you may need to install the Nagios::Plugin module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.34.0 /usr/local/share/perl/5.34.0 /usr/lib/x86_64-linux-gnu/perl5/5.34 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.34 /usr/share/perl/5.34 /usr/local/lib/site_perl) at /usr/local/lib/nagios/plugins/check_crm line 70.

Installing libmonitoring-plugin-perl manually fixed it, but I guess that should be set up by the charm

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2023-04-20T14:05:19Z
 * owner: [marcusboden](https://launchpad.net/~marcusboden)
 * assignee: [raychan96](https://launchpad.net/~raychan96)
 
 
 * the launchpad [url](https://bugs.launchpad.net/bugs/2017141)

**Comment by sudeephb** [comment]
(by [eric-chen](https://launchpad.net/~eric-chen))
I guess it is a duplicated issue with 
https://bugs.launchpad.net/charm-nrpe/+bug/2017602

**Comment by sudeephb** [comment]
(by [raychan96](https://launchpad.net/~raychan96))
Hi Marcus, thanks for reporting this issue. 

Can I ask what is your deployment? As far as I know charm-nrpe does not install check_crm on its own; it's provided by other principal charms. Principal charms that are related to charm-nrpe via nrpe-external-master interface, can request extra checks, and this is one of the possible situation that check_crm and the necessary packages get installed. For example, the deployment of 

- keystone:ha <-> hacluster:ha
- hacluster:nrpe-external-master <-> nrpe:nrpe-external-master 

will install `check_crm` and `check_corosync_rings` on keystone unit with proper required dependency "libmonitoring-plugin-perl" [1]. In this case, I was not able to reproduce your issue. 

[0] https://git.launchpad.net/charm-nrpe/tree/README.md#n67
[1] https://opendev.org/openstack/charm-hacluster/src/commit/0628e02983605313dd82b4501d54e9c027b669c7/hooks/hooks.py#L144

In short summary, your issue may not be related to the charm-nrpe but may be related to the principal charm itself. So it will be helpful if you can share your deployment.
